packages/arb-bridge-eth/contracts/PaymentRecords.sol

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019-2020, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pragma solidity ^0.5.3;
+
+contract PaymentRecords {
+	mapping(bytes32 => mapping(uint256 => mapping(address => address))) paymentMap;
```

Comment:
Should put a comment somewhere (either here or on getPaymentOwner) making clear that the presence of a record here does not imply that a payment exists.  A record means only that *if* such a payment exists, this is who owns it.